### ISSUE TITLE
Odd movement behaviour when e is remaped to j

### ISSUE THREAD

**Issue by LuyeT** [kyazdani42/nvim-tree.lua#367]
Hi, I'm running into an issue which I can't seem to identify the cause of. For context I've remapped some keys since I'm using colemak layout, namely 'neio' = 'hjkl' respectively.

The issue is that when I press 'e' while nvim-tree is focused, the first keypress of 'e' gets picked up by that statusline so I, for example, have to press eee to move the cursor down to lines.

This would make sense to me if there were some command beginning with 'e', but I don't see any indication of this from the documentation. Appreciate any suggestions on how I might fix this.

For reference here is all the remaps from my config file:

> "____________  COLEMAK keys ______________
>    noremap j o                                                                                                    
>    noremap J O                                                                                                    
>    noremap l i                                                                                                    
>    noremap L I                                                                                                    
>    noremap h e                                                                                                    
>    noremap H E                                                                                                    
>    noremap N K                                                                                                    
>    noremap E L                                                                                                    
>    noremap I H                                                                                                    
>    noremap O J                                                                                                    
>    noremap k n                                                                                                    
>    noremap K N                                                                                                    
>                                                                                                                   
> " movement                                                                                                      
>    noremap o l                                                                                                    
>    noremap i k                                                                                                    
>    noremap e j                                                                                                    
>    noremap n h                                                                                                    
>                                                                                                                   
>    noremap Y y$

**Comment by kyazdani42** [comment]
`:h nvim_tree_bindings` ?

**Comment by LuyeT** [comment]
Hi, just an update on this. It's not that the keybind wasn't working, I was able to move the cursor down, it's just that the commandline was picking up the first keypress, so I had to press it n+1 times. This issue went away once I cleaned away some old plugins I wasn't using, though I'm not sure which one was conflicting to be honest. Gonna close this issue as it's unlikely to be stritctly related to nvim-tree.
